wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -344,13 +334,23 @@ public void stopWorkspace(String workspaceId, Map<String, String> options)
 
     requireNonNull(workspaceId, "Required non-null workspace id");
     final WorkspaceImpl workspace = normalizeState(workspaceDao.get(workspaceId), true);
-    checkWorkspaceIsRunningOrStarting(workspace, "stop");
```

Comment:
I think we need this check because when workspace state is different to `STARTING` / `RUNNING`  we will update its attributes and try to stop it.

Reply:
I moved it back